lib/MusicBrainz/Server/WebService/Serializer/JSON/LD/Work.pm

```diff
@@ -45,7 +45,11 @@ around serialize => sub {
             $ret->{includedComposition} = list_or_single(map { serialize_entity($_->target, $inc, $stash) } @subworks);
         }
 
-        my @arrangements = grep { $_->direction == 1 } @{ $entity->relationships_by_link_type_names('arrangement') };
+        my @arrangements = (
+            (grep { $_->direction == 1 } @{ $entity->relationships_by_link_type_names('arrangement') }),
+            (grep { $_->direction == 2 } @{ $entity->relationships_by_link_type_names('medley') }),
```

Comment:
The relationship is "entity0 is a medley of entity1" - https://musicbrainz.org/relationship/c1dca2cd-194c-36dd-93f8-6a359167e992

We want to output entity0 as an arrangement of entity1 in entity1's JSON-LD. The direction is 2 when the source entity is entity1.